### PR TITLE
rename branch to main

### PR DESCRIPTION
**Pull request by mcharriere**
This PR renames ~app to `linkerd-control-plane`, matching upstream.~ main branch to `main`.

## Checklist

- [ ] Automated test are working (for chart changes)
- [ ] I am confident my changes don't break existing installations (for chart changes)
- [ ] Changelog entry has been added

### CODE REVIEW

File: tests/ats/README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ We want to ensure this app can be installed without any issues, so we're executi
 - Install [linkerd2-cni-app](https://github.com/giantswarm/linkerd2-cni-app) from giantswarm-test catalog
 - Install this chart
 - Install a test app ?
-- Execute `linkerd check --cni-namespace linkerd2-cni-app --linkerd-namespace linkerd2-app --output json --proxy --namespace test-app-namespace`
+- Execute `linkerd check --cni-namespace linkerd2-cni --linkerd-namespace linkerd --output json --proxy --namespace test-app-namespace`
```
Comment (Gacko) [inline tests/ats/README.md:12]: I know you only wanted to remove the `-app` suffix, but as far as I remember the CNI is running in `linkerd-cni`.

**Review by Gacko — COMMENTED** [review]
LGTM, hope you're able to clear this CI issue you mentioned in the issue.

File: tests/ats/README.md (hunk 1 above)
Comment (mcharriere) [reply tests/ats/README.md:12]: aha! that's right. thanks!

**Comment by mcharriere** [comment]
I discarded the app rename, as following upstream requires to change the repo name for our CI to work.

**Comment by Gacko** [comment]
Ok, still fine!

**Review by Gacko — APPROVED** [review]

**Pull request merged**